sdk/data/aztables/README.md

```diff
@@ -1,151 +1,415 @@
-Install the Azure Tables client library for Go :
+The Azure Data Tables SDK can access an Azure Storage or CosmosDB account.
 
 ### Prerequisites
-* An [Azure subscription][azure_sub].
-* An existing Azure storage account or Azure Cosmos DB database with Azure Table API specified.
-
-If you need to create either of these, you can use the [Azure CLI][azure_cli].
-
-#### Creating a storage account
-
-Create a storage account `mystorageaccount` in resource group `MyResourceGroup`
-in the subscription `MySubscription` in the West US region.
-
-
-#### Creating a Cosmos DB
+* Go versions 1.16 or higher
+* You must have an [Azure subscription][azure_subscription] and either
+    * an [Azure Storage account][azure_storage_account] or
+    * an [Azure Cosmos Account][azure_cosmos_account].
 
-Create a Cosmos DB account `MyCosmosDBDatabaseAccount` in resource group `MyResourceGroup`
-in the subscription `MySubscription` and a table named `MyTableName` in the account.
+#### Create account
+* To create a new storage account, you can use [Azure Portal][azure_portal_create_account], [Azure PowerShell][azure_powershell_create_account], or [Azure CLI][azure_cli_create_account]:
+* To create a new cosmos storage account, you can use the [Azure CLI][azure_cli_create_cosmos] or [Azure Portal][azure_portal_create_cosmos].
 
+### Install the package
+Install the Azure Data Tables client library for Python with `go get`:
+```bash
+go get github.com/Azure/azure-sdk-for-go/sdk/data/aztables
+```
+
+#### Create the client
+The Azure Data Tables library allows you to interact with two types of resources:
+* the tables in your account
+* the entities within those tables.
+Interaction with these resources starts with an instance of a [client](#clients). To create a client object, you will need the account's table service endpoint URL and a credential that allows you to access the account. The `endpoint` can be found on the page for your storage account in the [Azure Portal][azure_portal_account_url] under the "Access Keys" section or by running the following Azure CLI command:
+
+```bash
+# Get the table service URL for the account
+az storage account show -n mystorageaccount -g MyResourceGroup --query "primaryEndpoints.table"
+```
+
+Once you have the account URL, it can be used to create the service client:
+```golang
+cred, err := aztables.NewSharedKeyCredential("myAccountName", "myAccountKey")
+handle(err)
+serviceClient, err := NewServiceClient("https://<my_account_name>.table.core.windows.net/", cred, nil)
+handle(err)
+```
+
+For more information about table service URL's and how to configure custom domain names for Azure Storage check out the [official documentation][azure_portal_account_url]
+
+#### Types of credentials
+The clients support different forms of authentication. The aztables library supports any of the `azcore.TokenCredential` interfaces, authorization via a Connection String, or authorization with a Shared Access Signature Token.
```

Comment:
Does this mean you can use `azidentity.DefaultAzureCredential` and the like?

Reply:
I should clarify that you can use azidentity with a Storage account, but cosmos only supports SharedKeyCredential and SAS